### PR TITLE
Fix BiasBatchNorm2d layer and Pyramid implementation

### PR DESCRIPTION
**Pull request by eywalker**
* It appears that due to the update in PyTorch, previous implementation of `BiasBatchNorm2d` no longer works. I have updated the implementation to be cleaner and hopefully less dependent on the exact implementation of `nn.BatchNorm2d` and `nn.Module`
* The `Pyramid` implementation is now fixed to follow the standard Laplace image pyramid implementation. Some special parameters are added to maintain backward compatibility with the already trained networks using `Pyramid`.

### CODE REVIEW

File: attorch/layers.py [hunk 1]
```diff
@@ -755,16 +755,36 @@ def __repr__(self):
...
 
-
-class BiasBatchNorm2d(nn.BatchNorm2d):
+class BiasBatchNorm2d(nn.Module):
     def __init__(self, features, **kwargs):
         kwargs['affine'] = False
-        super().__init__(features, **kwargs)
-        self.bias = nn.Parameter(torch.Tensor(features))
+        super().__init__()
+        self.bn = nn.BatchNorm2d(features, **kwargs)
```
Comment (fabiansinz) [inline attorch/layers.py]: Shouldn't that have an `affine=False` by default then?

Comment (eywalker) [reply attorch/layers.py]: `affine=False` would allow the user to override it and also suggests that you can change the behavior. Given this, I prefer to simply fix the value in the init
